quickstarts/microsoft.logic/logic-app-create/metadata.json

```
@@ -1,41 +1,46 @@
 {
-  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
-  "icon": "logic",  
-  "itemDisplayName": "Create a Consumption logic app using an Azure Resource Manager template",
-  "description": "This template creates an empty logic app that can be used to define workflows.",
-  "summary": "Create a Consumption logic app with an ARM template.",
-  "githubUsername": "jeffhollan",
-  "docOwner": "ecfan",
-  "dateUpdated": "2022-07-28",
-  "type": "QuickStart",  
-  "tags": {
-    "subscriptiontype": "Azure4Student",
-    "id": 21,
-    "rank": 7,
-    "resource": "Logic Apps, MySQL",
-    "os": "Linux, Logic App",
-    "level": "beginner",
-    "cost": "$",
-    "scenario": "Azure Logic Apps",
-    "roles": "Developer",
-    "learning": "https://docs.microsoft.com/en-us/learn/browse/?products=azure-logic-apps&WT.mc_id=Portal-Microsoft_Azure_Education",
-    "isA4EActive": "true",
-    "usefulLink": [
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "icon": "logic",
+    "itemDisplayName": "Create a Consumption logic app using an Azure Resource Manager template",
+    "description": "This template creates an empty logic app that can be used to define workflows.",
+    "summary": "Create a Consumption logic app with an ARM template.",
+    "githubUsername": "jeffhollan",
```

Let's update the owner as Jeff has moved to bigger and better challenges. :)

```suggestion
    "githubUsername": "ecfan",
```